tests/unit/rpc/handlers/AllHandlerTests.cpp

```diff
@@ -105,6 +105,7 @@ struct AllHandlersDeathTest : HandlerBaseTest, testing::WithParamInterface<std::
     {
     }
 
+    StrictMockAmendmentCenterSharedPtr mockAmendmentCenterPtr;
```

Comment:
Maybe you could add an ASSERT in `initHandler()` to prevent the error in the future.